client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -400,7 +400,7 @@ export class DisplayTable extends React.Component {
                               } else {
                                 return col_configs_with_defaults[
                                   col_key
-                                ].formatter(total_row[col_key]);
+                                ].formatter(total_row[col_key], "white");
```

Comment:
I think we can safely make the text white across all display tables

so try putting `!important` on line 89 in Displaytable.scss. You'll probably need to work around with inline css on your formatters